stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -856,10 +620,7 @@ public final class _TensorComputation {
   // TODO(hongm): Retire returnValues when eager based runtime is removed.
   var returnValues: [CTensorHandle?]
 
-  // NOTE: the following properties are intentionally not implemented as an enum
-  // due to high churn, *please do not refactor for Swiftiness*.
-  private var stateTFE: TFEState?
-  private var stateTF: TFState?
+  private var stateTFE: TFEState
```

Comment:
If that's the only state, how about just `state`?
```suggestion
  private var state: TFEState
```

Reply:
done